core/app/assets/stylesheets/glass/layout/_pages.scss

```diff
@@ -0,0 +1,647 @@
+          }
+          &-visible {
+            display: flex;
+          }
+          &-tight-right {
+            right: -10px;
+          }
+          .circle-icon {
+            line-height: 1.6;
+            color: $cms-color;
+            border-color: $cms-color;
+            &:link,
+            &:visited {
+              color: $cms-color;
+            }
+            &:hover,
+            &:focus {
+              color: $white;
+              border-color: $cms-dark;
+              background: $cms-dark;
+            }
+            .icon-close {
+              font-size: 14px;
+            }
+          }
+          @media (min-width: 768px) {
+            .circle-icon {
+              color: $white;
+              border-color: $white;
+              &:link,
+              &:visited {
+                color: $white;
+              }
+            }
+          }
+          span {
+            display: inline-block;
+            width: 30px;
+            margin-left: 5px;
+            text-align: center;
+          }
+        }
+      }
+    }
+    /* nested ul */
+    ul {
+      margin: 0 0 0 0;
+      padding: 0;
+      list-style: outside none none;
+      &:empty {
+        margin: 0;
+      }
+      li {
+        margin-bottom: 0;
+        margin-left: 30px;
+        .page-row {
+          background: $white;
+        }
+        &:last-child {
+          .page-row {
+            margin-bottom: 4px;
+          }
+        }
+      }
+      li .page-row {
+        margin-left: -30px;
+        padding-left: 30px;
+      }
+      li li .page-row {
+        margin-left: -60px;
+        padding-left: 45px;
+      }
+      li li li .page-row {
+        margin-left: -90px;
+        padding-left: 60px;
+      }
+      li li li li .page-row {
+        margin-left: -120px;
+        padding-left: 75px;
+      }
+      li li li li li .page-row {
+        margin-left: -150px;
+        padding-left: 90px;
+      }
+      li li li li li li .page-row {
+        margin-left: -180px;
+        padding-left: 105px;
+      }
+      li.placeholder {
+        margin-left: 0;
+        padding-left: 30px;
+      }
+      li li.placeholder {
+        margin-left: -30px;
+        padding-left: 45px;
+      }
+      li li li.placeholder {
+        margin-left: -60px;
+        padding-left: 60px;
+      }
+      li li li li.placeholder {
+        margin-left: -90px;
+        padding-left: 75px;
+      }
+      li li li li li.placeholder {
+        margin-left: -120px;
+        padding-left: 90px;
+      }
+      li li li li li li.placeholder {
+        margin-left: -105px;
+        padding-left: 105px;
+      }
+    }
+  }
+  .placeholder {
+    box-sizing: border-box;
+    height: 50px;
+    margin-bottom: 2px;
+    padding: 10px 0 0 15px;
+    border: 1px dashed #cccccc;
+    background: lighten($off-white, 4%);
+    &:before {
+      position: relative;
+      top: 2px;
+      padding-left: 0;
+      font-family: "Refinery" !important;
+      font-size: 18px;
```

Comment:
I'll work on switching them to rem's. The relativity to the root font size makes things a lot more simple.